### ISSUE TITLE
classifies most sound as music

### ISSUE THREAD

**Issue by tc64** [IBM/audioset-classification#8]
When I pull data from audioset, the result is pretty good. But when I try to use my own data, it is very often classified as music. People talking, keyboard typing, dogs barking... I'm breaking the into 10 second clips, mapping to VGGish embeddings, and classifying them with a trained model. I've had success with one recording outside of audioset. 

I'm not sure if it has to do with properties of the audio like sample rate, background noise, or something else. Has anyone noticed this?

I'm using pytorch, decision_level_single_attention, trained for 50k iterations. The stats on validation set were comparable with what was reported. I'm using https://github.com/qiuqiangkong/audioset_classification/blob/master/extract_audioset_embedding/extract_audioset_embedding.py to convert sound files.

Thanks!

**Comment by tc64** [comment]
False alarm, I was not transforming the data using utils.utilities.transform_data.
